src/components/KeyFigure.vue

```diff
@@ -31,16 +31,21 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+@use '@/styles/typography';
+
 .keyFigure {
+  color: var(--color-text);
+  font-weight: 500;
+
   &__name {
     margin-bottom: 1rem;
+    font-size: typography.$font-size-1;
   }
```

Comment:
Burde denne også få med `letter-spacing` fra `title-4`-klassen i `typography`, sånn at den matcher andre overskrifter i den størrelsen? Eller er dette noe annet (men tilfeldigvis med samme størrelse og tykkelse som `title-4`)?

Reply:
Jo,`letter-spacing` burde den kanskje helst få med, ja. Bør kanskje bruke `title-4`-klassen, siden det vel er en form for tittel. For selve verdien har det kanskje ikke så mye hensikt å gjenbruke `title-1` 🤔